### PR TITLE
Fix background color not consistent

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Issue
Screenshot 2024-03-19 at 17 19 21

### Resolved
Screenshot 2024-03-19 at 17 19 03

### Demo

**Web**

https://github.com/linagora/flutter-date-range-picker/assets/160106668/4e726ca4-3752-48b6-9dbd-9515e0827d93

**Mobile**

https://github.com/linagora/flutter-date-range-picker/assets/160106668/8f07b90d-ba55-43af-a75a-c53de4ab4345

https://github.com/linagora/flutter-date-range-picker/assets/160106668/f9969696-8945-4a08-8626-5bc46c38dd54

### Additional fix: Single date picker background
**Before**
Screenshot 2024-03-27 at 17 19 51
**After**
Screenshot 2024-03-27 at 17 21 04
**Demo web**

https://github.com/linagora/flutter-date-range-picker/assets/160106668/92e02a4a-4444-49ee-8c18-87539c46b569

**Demo mobile**

https://github.com/linagora/flutter-date-range-picker/assets/160106668/9cc7c9fc-96e5-4375-949b-e63288312d7a

### CODE REVIEW

File: lib/material_date_range_picker_dialog.dart [hunk 1]
```diff
@@ -33,7 +33,7 @@ class MaterialDateRangePickerDialog {
       enableDrag: false,
       constraints: ResponsiveUtils.isLandscapeMobile(context)
         ? const BoxConstraints(maxWidth: 450)
-        : null,
+        : const BoxConstraints(maxWidth: 9999),
```
Comment (dab246) [inline lib/material_date_range_picker_dialog.dart]: Why set like that?

Comment (dab246) [reply lib/material_date_range_picker_dialog.dart]: What if the screen `size is > 9999`? (This latter example runs on TV)

Comment (tddang-linagora) [reply lib/material_date_range_picker_dialog.dart]: If constraint is null, the width of the dialog is limited, the bug color will appear. If constraint is any number and the screen's width is bigger than that number, the same bug will appear. I just made sure the width constraint is bigger than any available screen's width. As long as the constraint satisfies that condition, the actual number doesn't matter.

Comment (dab246) [reply lib/material_date_range_picker_dialog.dart]: How can you find the width limit of your screen? What if the width is greater than 9999?. IMO, there will be a better solution than this.

**Comment by dab246** [comment]
Please attach video demo for responsive web and mobile. Thanks

**Comment by tddang-linagora** [comment]
> Please attach video demo for responsive web and mobile. Thanks

Done

**Review by hoangdat — APPROVED** [review]

**Comment by hoangdat** [comment]
- [x] demo case focus in fields

**Comment by dab246** [comment]
Screenshot 2024-03-27 at 16 50 34

We should fix this function too.

**Comment by tddang-linagora** [comment]
> Screenshot 2024-03-27 at 16 50 34
> 
> We should fix this function too.

Done

**Review by dab246 — APPROVED** [review]
lgtm

**Comment by dab246** [comment]
- [x] Please rebase & squash commit

**Review by hoangdat — APPROVED** [review]

**Pull request merged**